src/ioctls/vcpu.rs

```diff
@@ -738,6 +1128,12 @@ impl VcpuFd {
             Err(io::Error::last_os_error())
         }
     }
+
+    /// Sets the `immediate_exit` flag on the `kvm_run` struct associated with this vCPU to `val`.
+    pub fn set_kvm_immediate_exit(&self, val: u8) {
```

Comment:
This feature relies on `KVM_CAP_IMMEDIATE_EXIT` being available. Can we add this cap as well as I believe that people using this functionality should also check the cap.

Reply:
All capabilities are defined here: https://github.com/rust-vmm/kvm-ioctls/blob/master/src/cap.rs

Reply:
@andreeaflorescu the capability is already defined in the current version of the crate [here](https://github.com/rust-vmm/kvm-ioctls/blob/2073af4101047422918a257481b36bcbd962a5ef/src/cap.rs#L137).

Can we merge?

Reply:
Missed that. Yap, I'll merge it.